### PR TITLE
chore - update library dependencies

### PR DESCRIPTION
**Pull request by amagdas**

### CODE REVIEW

File: superdesk/media/image.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def get_meta(file_stream):
             json.dumps(v)
             key = ExifTags.TAGS[k].strip()
 
-            if key == 'GPSInfo':
+            if key in 'GPSInfo':
```
Comment (petrjasek) [inline superdesk/media/image.py]: are you sure it does the same thing? :smiley:

Comment (amagdas) [inline superdesk/media/image.py]: Ha, I was supposed to revert the changes to this file :blush:

Comment (amagdas) [inline superdesk/media/image.py]: I wanted to upgrade to Pillow 3 but the metadata format changed  so I backpedaled :)

**Comment by petrjasek** [comment]
:+1:

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by mdhaman** [comment]
:+1:
